utils.py

```diff
@@ -4,9 +4,16 @@
 
 
 def send_email(to_email, subject, content, attachment=None):
-    EMAIL_INFO = json.loads(open('email.json').read())
+    with open('email.json') as f:
```

Comment:
Using "try catch" is better in case other developers do not have this file.

Reply:
Good idea, I will commit another version to avoid this.